R/filter_log_conditions.R

```diff
@@ -0,0 +1,89 @@
+# Filters a log list using args passed from `filter_log`, which
+# match args used in `compile_report`. This approaches list filtering
+# by unlisting to a vector, and matching by vector names and values.
+# Then the results of the filter are again filtered if `match_exact_type`
+# is specified. This could probably use a refactor to do all of this in
+# one step instead of two, but it works.
+
```

Comment:
Note this last sentence - "This could probably use a refactor to do all of this in
# one step instead of two, but it works."